Makefile

```
@@ -7,4 +7,14 @@ kind-create:
 	./scripts/init-kind-cluster.sh
 
 terraform-hack-init:
-	./hack/init.sh
\ No newline at end of file
+	./hack/init.sh
+
+create-rg:
+	kubectl apply -f ./examples/resourceGroup.yaml
+
+create-stor:
+	kubectl apply -f ./examples/storageAccount.yaml
+
+clear-crds:
```

```suggestion
clear-all:
```
Avoids confusion with deleting the CRD definitions themselves?